python/lsst/meas/astrom/matcher_probabilistic.py

```diff
@@ -79,8 +78,8 @@ class CatalogExtras:
 
     Parameters
     ----------
-    catalog : `pandas.DataFrame`
-        A pandas catalog to store extra information for.
+    catalog : `astropy.table.Table`
+        A Table to store extra information for.
```

Comment:
I know this is not the main point of the ticket, but the description of the catalog here is very unclear. A Table to store extra information for? It is ok not to change.